src/detectors/BTRK/BTRK.cc

```diff
@@ -21,7 +21,10 @@ void InitPlugin(JApplication *app) {
         "SiBarrelRawHits",
         {"SiBarrelHits"},
         {"SiBarrelRawHits"},
-        {}, // default config
+        {
+            .threshold = 0.65 * dd4hep::keV,
+	    .timeResolution = 10,
```

Comment:
So, this is right now a dangerous way to leave it: you add an explicit time resolution to digitization, but don't add one to reconstruction. Right now this might work because you set it to the default value. Someone might reasonably think that they can change the time resolution to something else. But that would put the time resolution used on digitization out of sync with the time resolution used in reconstruction. And that would give us incorrect reconstructed times.

Reply:
And that's not just happening here.